Mathlib/FieldTheory/SeparableClosure.lean

```diff
@@ -385,6 +391,38 @@ theorem sepDegree_bot' : sepDegree F (⊥ : IntermediateField E K) = sepDegree F
 theorem insepDegree_bot' : insepDegree F (⊥ : IntermediateField E K) = insepDegree F E :=
   insepDegree_eq_of_equiv _ _ _ ((botEquiv E K).restrictScalars F)
 
+variable (F) in
+lemma _root_.Field.insepDegree_top_le_insepDegree_of_isScalarTower :
```

Comment:
These lemmas popped out of nowhere, but they look good. Why are these three in the Field namespace though?

Reply:
Because `insepDegree` lives in namespace `Field`. I think all results on `sepDegree` and `insepDegree` are in namespace `Field`.

Reply:
Oh, I see. The last one is in the enclosing IntermediateField namespace.